src/NServiceBus.Transport.Msmq/MsmqConfigurationExtensions.cs

```diff
@@ -139,5 +139,86 @@ internal static bool GetShouldExecuteInstaller(this SettingsHolder settings)
             }
             return true;
         }
+
+        /// <summary>
+        /// Disables the storing of undeliverable messages in the dead letter queue
```

Comment:
Should we not also put a warning here? We do it in the other settings?

Reply:
Sure